cmd/start.go

```diff
@@ -51,14 +52,29 @@ var houseKeeperCmd = &cobra.Command{
 	Short: "starts ros-ocp housekeeper service",
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("starting ros-ocp housekeeper service")
-		services.StartHouseKeeperService()
+		sourcesFlag, _ := cmd.Flags().GetBool("sources")
+		partitionFlag, _ := cmd.Flags().GetBool("partition")
+		if sourcesFlag {
+			housekeeper.StartSourcesListenerService()
+		}
+		if partitionFlag {
+			housekeeper.DeletePartitions()
+		}
 	},
 }
 
+var sources bool
+var partition bool
+
 func init() {
 	rootCmd.AddCommand(startCmd)
 	startCmd.AddCommand(processorCmd)
 	startCmd.AddCommand(recommendationPollerCmd)
 	startCmd.AddCommand(apiCmd)
 	startCmd.AddCommand(houseKeeperCmd)
+
+	houseKeeperCmd.Flags().BoolVar(&sources, "sources", false, "starts sources listener service")
+	houseKeeperCmd.Flags().BoolVar(&partition, "partition", false, "deletes older partition")
+	houseKeeperCmd.MarkFlagsOneRequired("sources", "partition")
+	houseKeeperCmd.MarkFlagsMutuallyExclusive("sources", "partition")
```

Comment:
Do you think instead of flags to main command these can be subcommands?

Reply:
Yes it can be flag or the subcommand, depend on user preference.